docs/sass/_settings.scss

```diff
@@ -2,7 +2,6 @@ $breakpoint-medium: 900px;
 $grid-max-width: none;
 
 $content-padding: 0 5%;
-$sidebar-width: 18rem;
```

Comment:
This is still used in https://github.com/vanilla-framework/vanilla-framework/blob/1fcc0471ff0dccbaeded9cef47f72351c74f638f/docs/sass/styles.scss#L83 and gets redefined in https://github.com/vanilla-framework/vanilla-framework/blob/16048244cdea109353b4d912baf084aef284f5eb/scss/_patterns_code-numbered.scss#L2 . 

Second one shouldnt be a problem, but maybe another name at a later point?

Reply:
Explained in-person:
 - /docs has it's own nested, NPM-installed version of Vanilla so it can continue using the variable until it is upgraded to v2.0
 - the same-named variable is scoped to that component and is OK to stay